Description: Do not use namespaced oslo
 Package fails unit tests without this patch.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-02-08

--- python-oslo.concurrency-1.4.1.orig/oslo_concurrency/openstack/common/fileutils.py
+++ python-oslo.concurrency-1.4.1/oslo_concurrency/openstack/common/fileutils.py
@@ -19,7 +19,7 @@ import logging
 import os
 import tempfile
 
-from oslo.utils import excutils
+from oslo_utils import excutils
 
 LOG = logging.getLogger(__name__)
 
--- python-oslo.concurrency-1.4.1.orig/oslo_concurrency/processutils.py	2015-02-08 11:49:09.000000000 +0100
+++ python-oslo.concurrency-1.4.1/oslo_concurrency/processutils.py	2015-02-08 11:49:15.000000000 +0100
@@ -25,8 +25,8 @@
 import signal
 import time
 
-from oslo.utils import importutils
-from oslo.utils import strutils
+from oslo_utils import importutils
+from oslo_utils import strutils
 import six
 
 from oslo_concurrency._i18n import _
--- python-oslo.concurrency-1.4.1.orig/tests/test_warning.py	2015-02-08 12:06:47.000000000 +0100
+++ python-oslo.concurrency-1.4.1/tests/test_warning.py	2015-02-08 12:07:08.000000000 +0100
@@ -35,7 +35,7 @@
         with warnings.catch_warnings(record=True) as warning_msgs:
             warnings.resetwarnings()
             warnings.simplefilter('always', DeprecationWarning)
-            import oslo.concurrency
+            import oslo_concurrency
 
             # Use a separate function to get the stack level correct
             # so we know the message points back to this file. This
@@ -52,7 +52,7 @@
             # run the tests. A simpler test script run outside of
             # testr does correctly report the warnings.
             def foo():
-                oslo.concurrency.deprecated()
+                oslo_concurrency.deprecated()
 
             foo()
             self.assertEqual(1, len(warning_msgs))
